specs/market-api.yaml

```diff
@@ -687,7 +687,7 @@ paths:
             to the Agreement.
           - The Provider is now ready to accept a request to start an Activity
             as described in the negotiated agreement.
-          - The Requestor’s corresponding ConfirmAgreement call returns Ok after
+          - The Requestor’s corresponding `waitForApproval` call returns `Approved` after
```

Comment:
I assume that this is change, to be applied in future, because today both central and decentralized market return Ok

Reply:
Yes, this is for the future. Both need to comply with this spec.
But this is just a comment. Spec was merged within #34 already

Reply:
I've made a mistake. Sorry. Both impls are already returning `Approved`:
- central market always did
- decentralized changed y-day via https://github.com/golemfactory/yagna/pull/448